etc/helm/pachyderm/templates/kube-event-tail/podmonitor.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and .Values.kubeEventTail.enabled .Values.kubeEventTail.podMonitor }}
```

Comment:
I don't think we're typically providing PodMonitors; it's just for one particular distribution of prometheus which I'm not sure is widely used (prometheus-operator).  I think we should just skip this file entirely, and not make it an option.  If someone asks us for a pod monitor for kube-event-tail... easy to provide then ;)